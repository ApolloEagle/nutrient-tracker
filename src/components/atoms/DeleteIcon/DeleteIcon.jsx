import React from "react";
import { Delete } from "@mui/icons-material";

const DeleteIcon = () => <Delete />;

export default DeleteIcon;
