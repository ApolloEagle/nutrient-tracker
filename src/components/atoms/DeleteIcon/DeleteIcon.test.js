import React from 'react'
import { shallow } from 'enzyme';
import DeleteIcon from "./DeleteIcon";
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

const wrapper = shallow(<DeleteIcon />);

describe("Delete Icon", () => {
  it("Renders delete icon", () => {
    expect(wrapper).toMatchSnapshot();
  });
});