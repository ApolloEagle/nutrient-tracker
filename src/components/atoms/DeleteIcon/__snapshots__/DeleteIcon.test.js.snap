// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Delete Icon Renders delete icon 1`] = `<Memo(ForwardRef(DeleteIcon)) />`;

exports[`Delete icon unit test Renders delete icon 1`] = `<Memo(ForwardRef(DeleteIcon)) />`;
